src/test/java/org/cactoos/scalar/ScalarWithFallbackTest.java

```diff
@@ -96,39 +110,39 @@ public void usesFallbackOfInterruptedException() throws Exception {
                     )
                 )
             ),
-            new ScalarHasValue<>(message)
+            new IsEqual<>(message)
         );
     }
 
     @Test
     public void usesTheClosestFallback() throws Exception {
-        final String expected = "Fallback from IllegalFormatException";
-        MatcherAssert.assertThat(
+        final String expected = "Fallback from IllegalStateException";
+        new Assertion<>(
             "Can't find the closest fallback",
             new ScalarWithFallback<>(
                 () -> {
-                    throw new IllegalFormatWidthException(1);
+                    throw new IllegalStateException("wat is raw");
```

Comment:
@scratchboom What's "wat"?

Reply:
Sorry it's just left from my experiments. This string is not necessary). Removed and pulled.

Reply:
@scratchboom Right.